Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM golang:1.13 as builder 
 
 WORKDIR /go/src/github.com/hobbyfarm/gargantua
-COPY . .
+COPY go.mod go.sum ./
+RUN go mod download
 
 ENV GOOS=linux 
 ENV CGO_ENABLED=0
 
-RUN go get -d -v ./...
-RUN go install -v ./...
+COPY . .
+RUN go install -v
```

Comment:
we checkin the vendors now since go modules, should probably pass in -mod vendor and get rid of all the downloads shouldnt we?

Reply:
Is there a good reason to keep the `vendor` directory? It doesn't appear to be necessary.

Reply:
ask 6 different devs and you'll get 7 different answers. i personally like it because it's safer and you have everything the minute you clone

Reply:
Sounds about right :laughing: I've updated the build to use what's already in the repo.